sdk/identity/azure-identity/pom.xml

```diff
@@ -37,13 +37,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>msal4j-persistence-extension</artifactId>
-      <version>0.2.0</version> <!-- {x-version-update;com.microsoft.azure:msal4j-persistence-extension;external_dependency} -->
-      <exclusions>
-        <exclusion>
-          <groupId>org.slf4j</groupId>
```

Comment:
so, the exclusion is removed because it is excluded from msal persistence extension library ?

Reply:
Yes it was a bug in the last release and they have removed it